go/mysql/flavor_mariadb.go

```diff
@@ -154,6 +155,25 @@ func (mariadbFlavor) status(c *Conn) (SlaveStatus, error) {
 	if err != nil {
 		return SlaveStatus{}, vterrors.Wrapf(err, "SlaveStatus can't parse MariaDB GTID (Gtid_Slave_Pos: %#v)", resultMap["Gtid_Slave_Pos"])
 	}
+	filePos, err := strconv.Atoi(resultMap["Exec_Master_Log_Pos"])
```

Comment:
I think we should tolerate these values being empty, and leave the corresponding position field at its zero value if so. If we pass an empty string to `strconv.Atoi`, it will return an error.

Reply:
Good idea. I didn't realize empty string resulted in an error.

Reply:
Done.